### PR TITLE
Add cleanup script for clearing out any sensitive access/data

### PR DESCRIPTION
**Pull request by alexsoble**
# Who is this PR for?

Developers who have Heroku access and/or occasionally need to work with some production data locally 

# What problem does this PR fix?

Adds a simple-to-use and thorough command to clean up your workspace and clear it of all sensitive data/access when you're done

# What does this PR do?

### Command line 

```
./scripts/cleanup/cleanup.sh  
```

### Sample output 

```
No /tmp folder to delete...

No /data folder to delete...

Deleting local_env.yml...

Logging out of Heroku...

Local credentials cleared
```

### CODE REVIEW

**Comment by alexsoble** [comment]
cc @kevinrobinson — this is a little script I've been using for when I finish up a work session that used real data locally or involved Heroku access, you might find it handy too although I know we often use different workflows when developing/testing locally. 

Merging this now mostly as a tool for myself, if you want to use it too and/or have suggestions for improvements those are both great. 🎉

**Comment by alexsoble** [comment]
Oooh I suppose I should add a note about this to the README

File: scripts/cleanup/cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  echo
+fi
+
+echo "Logging out of Heroku..."; echo
+heroku logout
+echo
+
+echo "Resetting the database..."; echo
+rake db:drop
+rake db:create
```
Comment (kevinrobinson) [inline scripts/cleanup/cleanup.sh]: Is this in case you load production data locally?

**Comment by kevinrobinson** [comment]
@alexsoble This is awesome! 🌮 👍

**Comment by alexsoble** [comment]
@kevinrobinson Thanks! Yeah the db drop/create is in case you load
production data locally.

**Pull request merged**